src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -121,16 +121,30 @@
 				<ul class="h-100" :class="{'scroller': isSearching}">
 					<!-- Conversations List -->
 					<template v-if="!isSearching">
+						<NcAppNavigationCaption v-if="isFiltered && filteredConversationsList.length > 0" :name="getFilterCaption" />
+						<NcEmptyContent v-else-if="initialisedConversations && filteredConversationsList.length === 0"
+							class="scroller"
+							:name="t('spreed', 'No matches found')"
+							:description="messageDisplay">
+							<template #icon>
+								<MessageBadge :size="64" />
+							</template>
+							<template #action>
+								<NcButton @click="handleFilter(null)">
+									<template #icon>
+										<FilterRemoveIcon :size="20" />
+									</template>
+									{{ t('spreed', 'Clear filter') }}
+								</NcButton>
+							</template>
+						</NcEmptyContent>
 						<li class="h-100">
```

Comment:
```suggestion
						<li v-show="filteredConversationsList.length > 0" class="h-100">
```
If there is nothing to show, component is still there, and adds a scroll